pages/tutorials/[slug].js

```diff
@@ -77,6 +78,11 @@ export default function Tutorial({ name, steps, id, slug }) {
             {currentStep === 0 ? "start" : "next"}
           </Button>
         )}
+        {currentStep === steps.length && (
+          <Link href="/tutorials">
+            <Button isPrimary={true}>back to library</Button>
```

Comment:
you don't need `{true}`. If you pass only the prop without a value it is automatically a Boolean(true)